Cue/app/common/DeckPreviewHeader.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+
+'use strict';
+
+import React from 'react'
+import { View, Text, Platform } from 'react-native'
+
+import Button from 'react-native-button'
+
+import type { DeckMetadata } from '../api/types'
+
+import CueColors from './CueColors'
+
+import CueTabs from './CueTabs'
+import DeckRating from './DeckRating'
+
+const styles = {
+  container: {
+    backgroundColor: CueColors.primaryTint,
+    elevation: 4
+  },
+  textContainer: {
+    paddingHorizontal: 16,
+    paddingBottom: 16
+  },
+  deckTitle: {
+    color: 'white',
+    fontSize: 28
+  },
+  deckAuthor: {
+    color: 'white',
+    fontSize: 12
+  },
+  ratings: {
+    color: 'white',
+  },
+  tabs: {
+    marginTop: 20
+  },
+  addButton: {

```

Comment:
The button actually doesn't show up since it's blue-on-blue, and the alignment is off due to how the font is rendered.

Try this?

```js
  addButton: {
    color: 'white',
    borderColor: 'white',
    borderWidth: 1,
    borderRadius: 4,
    fontSize: 13,
    fontWeight: '600',
    paddingTop: 6,
    paddingBottom: 4,
    paddingHorizontal: 8,
  },
```

Looks like:
<img width="383" alt="screen shot 2017-03-15 at 9 59 50 pm" src="https://cloud.githubusercontent.com/assets/13400887/23978506/bd3106d0-09ca-11e7-8e86-71360a426dcd.png">